docs/user/04-metrics.md

```diff
@@ -22,13 +22,13 @@ Optionally, the Telemetry module provides a DaemonSet of an [OTel Collector](htt
 1. An application exposing metrics in OTLP, pushes metrics to the central metric gateway service.
```

Comment:
Consistency: In the diagram, either use 3rd person ("discovers", "pushes", "collects"...) or infinitive ("pull"), but don't mix them.

Reply:
done